install/upgrade_to_2.13.7.php

```diff
@@ -45,6 +47,7 @@ public function isResyncIssuesRequired() {
    public function upgrade(Migration $migration) {
       $this->migration = $migration;
       $this->fixEncodingInQuestions();
+      $this->resizeWidgets();
```

Comment:
It seems simpler and less risky not to resize at all at the update, neither in this version nor in 2.13.7.